src/assets/arrays/theme.ts

```diff
@@ -0,0 +1,39 @@
+
+export const colors = {
+    darkGrey: '#353d5e',
+    lightGrey: '#F5F5F5',
+    white: 'white',
+    transparentdarkBg: 'rgba(0,0,0,0.5)',
+    buttonBglight: 'rgba(255, 99, 71, 0.5)',
+    buttonBgdark: 'rgb(60, 60, 60)',
+    yellow: '#FFDB58',
+    coral: '#E77471',
+    darkbg: 'rgba(0,0,0,0.3)',
+    mediumGrey: '#B0B0B0',
+    coralbg: 'rgba(255, 99, 71, 0.1)',
+    pink: 'hsla(0, 100%, 30%, 0.3)',
+    offwhite : '#FDF7EE',
+    transparentlightBg: 'rgba(240,240,240,0.8)',
+    red: '#F67280',
+}
+
+export const images = {
+    logo: require('/Users/coditas/Desktop/shop/src/assets/images/nida.png'),
```

Comment:
Will this path work on a different system?
even if it is a relative import, Path should be within the project scope. Not relative to the user directory